test/cli/project-init/missing-params/expected-stderr

```diff
@@ -1,3 +1,2 @@
 Error:
 - Missing Storage Api host, please use "--storage-api-host" flag or ENV variable "KBC_STORAGE_API_HOST".
-- Missing Storage Api token, please use "--storage-api-token" flag or ENV variable "KBC_STORAGE_API_TOKEN".
```

Comment:
Kedze sa uz `host` (`dependencies.ForLocalCommand`)  a `token` (`dependencies.ForRemoteCommand`)  nacitavaju na roznych miestach, tak uz nie je moznost tieto chyby spojit (bez duplicitneho kodu).

Urcite by bolo lepsie vypisat tie chyby naraz, ale nedava to zmysel vzhladom na potrebnu namahu.
Mozno sa to vylepsi, pri niektorom dalsom refactoringu.